submissions/description/server_security_misconfiguration/misconfigured_dns/template.md

```diff
@@ -0,0 +1,21 @@
+# Misconfigured DNS
+
+## Overview of the Vulnerability
+
+Domain Name System (DNS) misconfigurations can result in internal information being leaked publicly. The DNS misconfiguration identified on the endpoints allows an attacker to view the internal network structure of the domain, transfer the zone file from one DNS to . From here, the attacker can use this information to form other attacks on the network and its users.
```

Comment:
Unsure of this wording and what it means
> transfer the zone file from one DNS to .

Reply:
Thanks @evildaemond, had a D'oh moment writing that sentence...